geoengine/workflow.py

```diff
@@ -438,3 +430,64 @@ def workflow_by_id(workflow_id: UUID) -> Workflow:
     # TODO: check that workflow exists
 
     return Workflow(WorkflowId(workflow_id))
+
+
+class Colorizer:
+    '''
+    Creates a colormap and returns its JSON string
+    '''
+
+    __colormap_name: str
+    __colorizer_min_max: Tuple[float, float]
+    __steps: int
+    __no_data_color: Tuple[float, float, float, float]
+
+    def __init__(self,
+                 colorizer_min_max: Tuple[float, float],
+                 colormap_name: str = "viridis",
```

Comment:
Maybe you can call `get_cmap` already in the constructor and fail / throw an error if this colorizer does not exist.